src/test/java/org/example/ManWomanTest.java

```diff
@@ -0,0 +1,169 @@
+package org.example;
+
+import org.example.exception.*;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+public class ManWomanTest {
+
+    private Man man = null;
+    private Woman woman = null;
+    private Man retiredMan = null;
+    private Woman retiredWoman = null;
+
+
+    @BeforeMethod
+    public void beforeMethod() throws Exception {
+        man = new Man("John", "Smith", 23);
+        woman = new Woman("Mary", "Adams", 21);
+        retiredMan = new Man("Tom", "Cat", 66);
+        retiredWoman = new Woman("Jerry", "Mouse", 61);
```

Comment:
You do not need before and after methods here, you can create objects inline near definition of class variables